lib/SPIRV/SPIRVInternal.h

```diff
@@ -398,6 +398,7 @@ const static char MaxWGSize[] = "max_work_group_size";
 const static char NoGlobalOffset[] = "no_global_work_offset";
 const static char MaxWGDim[] = "max_global_work_dim";
 const static char NumSIMD[] = "num_simd_work_items";
+const static char FmaxMhz[] = "scheduler_target_fmax_mhz";
```

Comment:
A couple of minor comments/questions:
 - strictly speaking, not all of metadata names in this namespace are _SPIR_ metadata per https://www.khronos.org/registry/SPIR/specs/spir_spec-2.0.pdf. Should we move non-SPIR metadata names to a separate namespace?
 - should we have `intel` prefix or suffix for metadata introduced by Intel vendor extensions?

Reply:
>> Should we move non-SPIR metadata names to a separate namespace?

That is a good suggestion. Let me do it in a separate patch.

>> should we have intel prefix or suffix for metadata introduced by Intel vendor extensions?

At first I was about to strongly agree with you (with a remark, that metadata names come from backends' developers, hence we need to align with them). Now I still agree with you, but also I think, that some of these names can become not only Intel-specific (in a certain circumstances). For example ivdep pragma/attribute is quiet popular in among compiler, see: https://gcc.gnu.org/onlinedocs/gcc-4.9.2/gcc/Loop-Specific-Pragmas.html .